concent_api/core/tests/test_unit_admin_tags.py

```diff
@@ -0,0 +1,150 @@
+import datetime
+import hashlib
+from assertpy import assert_that
+from freezegun import freeze_time
+from golem_messages.message.concents import ForceGetTaskResult
+from golem_messages.factories.tasks import ReportComputedTaskFactory
+from common.helpers import get_current_utc_timestamp
+from core.message_handlers import store_subtask
+from core.models import Subtask
+from core.templatetags.admin_tags import get_longest_lasting_subtask_timestamp, get_time_until_concent_can_be_shut_down
+from core.tests.utils import ConcentIntegrationTestCase
+from core.utils import hex_to_bytes_convert
+from api_testing_common import create_signed_task_to_compute
+from api_testing_common import timestamp_to_isoformat
+
+
+def store_report_computed_task_as_subtask(current_time, task_id, subtask_id, deadline, next_deadline, subtask_state):
+    file_content = '1'
+    file_size = len(file_content)
+    file_check_sum = 'sha1:' + hashlib.sha1(file_content.encode()).hexdigest()
+    with freeze_time(timestamp_to_isoformat(current_time - 100)):
+        task_to_compute = create_signed_task_to_compute(
+            task_id=task_id,
+            subtask_id=subtask_id,
+            deadline=deadline,
+            price=0,
+        )
+    with freeze_time(timestamp_to_isoformat(current_time - 10)):
+        report_computed_task = ReportComputedTaskFactory(
+            task_to_compute=task_to_compute,
+            size=file_size,
+            package_hash=file_check_sum,
+            subtask_id=subtask_id,
+        )
+    force_get_task_result = ForceGetTaskResult(
+            report_computed_task=report_computed_task,
+    )
+    store_subtask(
+        task_id=report_computed_task.task_to_compute.task_id,
+        subtask_id=report_computed_task.task_to_compute.subtask_id,
+        provider_public_key=hex_to_bytes_convert(report_computed_task.task_to_compute.provider_public_key),
+        requestor_public_key=hex_to_bytes_convert(report_computed_task.task_to_compute.requestor_public_key),
+        state=subtask_state,
+        next_deadline=next_deadline,
+        task_to_compute=report_computed_task.task_to_compute,
+        report_computed_task=report_computed_task,
+        force_get_task_result=force_get_task_result,
+    )
+
+
+def store_force_get_task_result(current_time, deadline, next_deadline, task_id, subtask_state):
+    subtask_id = task_id + '1'
+
+    store_report_computed_task_as_subtask(
+        current_time,
+        task_id,
+        subtask_id,
+        deadline,
+        next_deadline,
+        subtask_state,
+    )
+
+
+class TestAdminTagsQuerySet(ConcentIntegrationTestCase):
+    @staticmethod
```

Comment:
This can be removed if you use `self.assert()` instead of `assert_that()`. It is ok for me how it is defined now but I would like other guys to speak up.
@cameel @dybi ?

Reply:
For me: do whatever you want, but don't make tests `@staticmethods`.
I am fine with both:
1. `pylint` suppression
2. changing the assertion

Reply:
BTW: `store...` functions can be moved to test class (and then they will be called with `self)`